screenpipe-server/src/bin/screenpipe-server.rs

```diff
@@ -15,7 +15,7 @@ use screenpipe_core::find_ffmpeg_path;
 use screenpipe_server::{
     cli::{Cli, CliAudioTranscriptionEngine, CliOcrEngine, Command, PipeCommand}, start_continuous_recording, watch_pid, DatabaseManager, PipeControl, PipeManager, ResourceMonitor, Server, highlight::{Highlight,HighlightConfig}
 };
-use screenpipe_vision::monitor::list_monitors;
+use screenpipe_vision::{monitor::list_monitors, run_ui};
```

Comment:
the import should be in 

#cfg macos 

like the rest of it, see linux build complaining:

<img width="275" alt="image" src="https://github.com/user-attachments/assets/855c88a4-258d-4fde-82c2-c754023451cf">